frontend/server/app.ts

```diff
@@ -108,9 +109,9 @@ function createUIServer(options: UIConfigs) {
   /** Healthz */
   registerHandler(
     app.get,
-    `/${apiVersionPrefix}/healthz`,
+    `/${apiVersion1Prefix}/healthz`,
     getHealthzHandler({
-      healthzEndpoint: getHealthzEndpoint(apiServerAddress, apiVersionPrefix),
+      healthzEndpoint: getHealthzEndpoint(apiServerAddress, apiVersion1Prefix),
       healthzStats: getBuildMetadata(currDir),
     }),
```

Comment:
Why are these handlers for `apiVersion1Prefix` only but not `apiVersion2Prefix`?

Reply:
Added another one for v2